package.json

```diff
@@ -21,37 +21,38 @@
   "scripts": {
     "lint": "eslint .",
     "pretest": "npm run lint",
-    "test": "nyc mocha --async-only",
-    "azure-pipelines": "nyc mocha --async-only --reporter xunit -O output=test.xunit",
-    "coveralls": "nyc report --reporter=text-lcov | coveralls"
+    "test": "nyc mocha --async-only"
   },
   "dependencies": {
-    "arr-flatten": "^1.0.1",
-    "arr-map": "^2.0.0",
-    "bach": "^1.0.0",
-    "collection-map": "^1.0.0",
-    "es6-weak-map": "^2.0.1",
-    "last-run": "^1.1.0",
-    "object.defaults": "^1.0.0",
-    "object.reduce": "^1.0.0",
-    "undertaker-registry": "^1.0.0",
-    "fast-levenshtein": "^1.0.0"
+    "bach": "^1.2.0",
+    "fast-levenshtein": "^3.0.0",
+    "last-run": "^2.0.0",
+    "undertaker-registry": "^2.0.0"
   },
   "devDependencies": {
-    "async-once": "^1.0.0",
-    "coveralls": "github:phated/node-coveralls#2.x",
-    "del": "^2.0.2",
-    "eslint": "^2.13.1",
-    "eslint-config-gulp": "^3.0.1",
-    "expect": "^1.20.2",
-    "gulp-jshint": "^1.8.4",
-    "mocha": "^3.0.0",
-    "nyc": "^10.3.2",
-    "once": "^1.3.1",
-    "through2": "^2.0.0",
-    "undertaker-common-tasks": "^1.0.0",
-    "undertaker-task-metadata": "^1.0.0",
-    "vinyl-fs": "^2.2.0"
+    "async-once": "^2.0.0",
+    "del": "^6.1.1",
+    "eslint": "^7.32.0",
+    "eslint-config-gulp": "^5.0.1",
+    "expect": "^27.5.1",
+    "gulp-jshint": "^2.1.0",
+    "jshint": "^2.13.5",
```

Comment:
```suggestion
```
What are these used for? They should be removed

Reply:
@phated 
`gulp-jshint` is used [here](https://github.com/sttk/undertaker/blob/chore-normalize-repository/test/integration.js#L61) in test.

Reply:
I will probably rewrite the test to not rely on a plugin like this.